kubectl_plugins/create-service-broker/create-service-broker.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/kubernetes-incubator/service-catalog/kubectl_plugins/utils"
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `
+
+INVALID USAGE:
+
+Usage:
+  kubectl plugin create-service-broker BROKER_NAME BROKER_URL
```

Comment:
I will do this in a follow up, however, since option flag is not yet available, the syntax might not be as good as we want. We could have 1 single optional argument and everything (namespace/secret..) will have to go into that argument.